extensions/arc/src/types.ts

```diff
@@ -15,6 +15,14 @@ export type Tab = {
 
 export type TabLocation = "topApp" | "pinned" | "unpinned";
 
+export function isTab(tab: Tab): tab is Tab {
```

Comment:
I don't think this function should be located in a file strictly created to house types

I'd recommend moving it to the utils.ts file

Reply:
@hugivar done ✅